contracts/modules/STO/STO.sol

```diff
@@ -55,4 +55,20 @@ contract STO is ISTO, STOStorage, Module {
     function _getKey(bytes32 _key1, address _key2) internal pure returns(bytes32) {
         return bytes32(keccak256(abi.encodePacked(_key1, _key2)));
     }
+
+    function _allowPreMinting(uint256 _tokenAmount) internal {
```

Comment:
Should we allow users to turn this off after turning it on? If so, I guess we'd either have to move the already issued to the treasury wallet, or burn them (I'd favour the latter).

Reply:
I don't think this is necessary, It will add complexity in the code (+ code size) nothing else. The issuer has the clear choice of where to go at the time of launch of the STO either pre-mint option or mint on buying option.

Reply:
Checking w/ Product on this.